src/modules/import.spec.js

```diff
@@ -1,25 +1,26 @@
 import reducer, {
-  STARTED,
-  CANCELED,
-  PROGRESS,
-  FINISHED,
-  FAILED,
+  // STARTED,
+  // CANCELED,
+  // PROGRESS,
+  // FINISHED,
+  // FAILED,
   FILE_TYPE_SELECTED,
   selectImportFileType,
-  FILE_SELECTED,
+  // FILE_SELECTED,
   selectImportFileName,
-  OPEN,
-  CLOSE,
-  SET_PREVIEW,
-  SET_DELIMITER,
-  SET_GUESSTIMATED_TOTAL,
-  SET_STOP_ON_ERRORS,
-  SET_IGNORE_BLANKS,
-  TOGGLE_INCLUDE_FIELD,
-  SET_FIELD_TYPE,
+  // OPEN,
+  // CLOSE,
+  // SET_PREVIEW,
+  // SET_DELIMITER,
+  // SET_GUESSTIMATED_TOTAL,
+  // SET_STOP_ON_ERRORS,
+  // SET_IGNORE_BLANKS,
+  // TOGGLE_INCLUDE_FIELD,
+  // SET_FIELD_TYPE,
```

Comment:
Currently are not used and fail `npm run check` hook. I don't want to delete them completely because Lucas has lots of unfinished code here and we might want to use it later to understand initial module architecture.